perl/p5-hash-moreutils/Portfile

```diff
@@ -4,18 +4,16 @@ PortSystem          1.0
 PortGroup           perl5 1.0
 
 perl5.branches      5.26
-perl5.setup         Hash-MoreUtils 0.05
+perl5.setup         Hash-MoreUtils 0.06
 license             {Artistic-1 GPL}
 maintainers         nomaintainer
 description         Hash::MoreUtils - Provide the stuff missing in Hash::Util
 long_description    ${description}
 
 platforms           darwin
 
-checksums           rmd160  8d30e017153bfcfe38405412e756596c02725da7 \
-                    sha256  5e9c8458457eb18315a5669e3bef68488cd5ed8c2220011ac7429ff983288ab1
+checksums           rmd160  6c24a4fbdab3b05033f80a0d2e7369ab2e2fbfe5 \
+                    sha256  db9a8fb867d50753c380889a5e54075651b5e08c9b3b721cb7220c0883547de8 \
+                    size    9208
 
-if {${perl5.major} != ""} {
-    perl5.use_module_build
-    supported_archs noarch
-}
```

Comment:
I know little or nothing about the perl5 portgroup so I don't know if this is correct.

Reply:
I removed `perl5.use_module_build` because the module has switched to using ExtUtils::MakeMaker in this version.

I don't know if `supported_archs noarch` belongs inside this `if` block, since some ports put it in one while others don't. I asked about that on IRC earlier today, but didn't get a reply to that specifically.

Reply:
Gotcha. For now, I'd just remove `perl5.use_module_build` as that's the most conservative possible change.

Reply:
Alright then, done in 3c58869258739973dc52d555ca89271f1adf9047